bot_impl.py

```diff
@@ -4331,17 +4331,14 @@ async def on_message(message):
                     await safe_send(message.author, "You don't have permission to end the game.")
                     return
 
-                if argument.lower() != "good" and argument.lower() != "evil":
-                    await safe_send(message.author, "The winner must be 'good' or 'evil' exactly.")
+                if argument.lower() != "good" and argument.lower() != "evil" and argument != "tie":
```

Comment:
argument.lower()

Reply:
now calling `argument = argument.lower()` above to not need to call it in four places.